vendor/bat-native-ads/src/bat/ads/internal/ml/pipeline/pipeline_embedding_info.cc

```diff
@@ -0,0 +1,38 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/ml/pipeline/pipeline_embedding_info.h"
+
+#include <utility>
+
+namespace ads {
+namespace ml {
+namespace pipeline {
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo() = default;
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo(
+    PipelineEmbeddingInfo&& info) noexcept = default;
+
+PipelineEmbeddingInfo& PipelineEmbeddingInfo::operator=(
+    PipelineEmbeddingInfo&& info) noexcept = default;
+
+PipelineEmbeddingInfo::~PipelineEmbeddingInfo() = default;
+
+PipelineEmbeddingInfo::PipelineEmbeddingInfo(
+    const int version,
+    const std::string& timestamp,
+    const std::string& locale,
+    const int embeddings_dim,
```

Comment:
can we infer this argument from the following argument?

Reply:
Removed embeddings_dim and associated references (it was just a carryover from a different resource)

Reply:
Having embedding_dim pre-computed saves time, since it is needed every time a new vector is initialized